server/service/src/invoice_line/inbound_shipment_service_line/update/generate.rs

```diff
@@ -31,7 +32,7 @@ pub fn generate(
         None
     };
 
-    let mut update_line = existing_line;
+    let mut update_line = existing_line.clone();
```

Comment:
nit: Why do you need to clone it now? can it be avoided?

Reply:
Before I had some ownership problem but can add it back now with the changes